CRM/Certificate/Token/Certificate.php

```diff
@@ -65,16 +71,63 @@ public function prefetch(TokenValueEvent $e) {
     return $resolvedTokens;
   }
 
+  private function getMembershipDates(TokenValueEvent $e): array {
```

Comment:
Hi, @shahrukh-compuco Given that these new tokens are related to the membership entity, I believe they are better moved to and resolved in the membership token class https://github.com/compucorp/uk.co.compucorp.certificate/blob/master/CRM/Certificate/Token/Membership.php

Reply:
Hey Ola
As discussed that there might be a new ticket from Pallavi to transfer all the work to membership type only, I think then not only these but other work in this workstream will be transffered at once to the membership classes